tests/apps/search/test_indexers_organizations.py

```diff
@@ -121,6 +121,22 @@ def test_indexers_organizations_get_es_documents(self, _mock_picture):
             ],
         )
 
+    def test_indexers_organizations_get_es_documents_unpublished(self):
+        """
+        Unpublished categories and children of unpublished categories should not be indexed
+        """
+        OrganizationFactory()
+
+        # The unpublished organization should not get indexed
+        self.assertEqual(
+            list(
+                OrganizationsIndexer.get_es_documents(
+                    index="some_index", action="some_action"
+                )
+            ),
+            [],
+        )
+
```

Comment:
Maybe add a similar test for courses to secure all possibilites ?

Reply:
True. I assumed it was already tested... but no!